zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -63,10 +63,17 @@ alert.api.action.updateAlert.param.references = The reference details for the al
 alert.api.action.updateAlert.param.evidence = The evidence associated with the alert.
 alert.api.action.updateAlert.param.cweId = The CWE identifier associated with the alert.
 alert.api.action.updateAlert.param.wascId = The WASC identifier associated with the alert.
+alert.api.action.updateAlertsConfidence = Update the confidence of the alerts with the given IDs (comma separated values).
```

Comment:
Just `Update the confidence of alerts.`? To avoid repeating the description of the parameter (same for the other endpoint).